src/components/Player/GridControls.js

```diff
@@ -210,6 +210,9 @@ export default class GridControls extends Component {
         this.selectNextClue(true);
       } else if (key === 'i') {
         onVimInsert && onVimInsert();
+      } else if (key === 's') {
+        this.handleAction('delete', shiftKey);
```

Comment:
i think shiftKey isn't necessary for delete -- it takes no params in https://github.com/dangeng/downforacross.com/blob/master/src/components/Player/GridControls.js#L322

Reply:
actually it would probably be cleanest to just do `this.delete()` instead of going through `handleAction`.